src/metabase/driver/druid/query_processor.clj

```diff
@@ -164,7 +165,8 @@
   {:pre [(string? format-str)]}
   {:type     :timeFormat
    :format   format-str
-   :timeZone "US/Pacific" ; TODO - should we use `report-timezone` instead (?)
+   :timeZone (or (driver/report-timezone)
+                 "UTC")
```

Comment:
This should be using the `:settings` already provided on the expanded query via https://github.com/metabase/metabase/blob/master/src/metabase/query_processor.clj#L114-L126

That was put in there so that individual drivers don't need to lookup and worry about extracting properties like this and instead and just access them on the query directly